tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -213,4 +213,21 @@ DRAW_TEST_F(CGContext, AntialiasToggleTranscendsGState, WhiteBackgroundTest<>) {
         { 15, 5 }, { bounds.size.width, bounds.size.height - 10 },
     };
     CGContextStrokeLineSegments(context, blurryLineSegments2, 2);
-}
\ No newline at end of file
+}
+
+DRAW_TEST_F(CGContext, NullFillColor, WhiteBackgroundTest<>) {
```

Comment:
We only need to test that the fill/stroke colors aren't changed when nullptr is passed in correct? Is there a way to get the color from the context to check?